lib/website_templates/adf_diag.css

```diff
@@ -5,71 +5,94 @@ nav.primary-navigation {
   text-align: center;
   font-size: 16px;
 }
-nav.primary-navigation ul li {
+
+nav.primary-navigation ul{
   list-style: none;
-  margin: 0 auto;
-  border-left: 2px solid #3ca0e7;
-  border-right: 2px solid #3ca0e7;
+  margin: 0;
+  padding: 0;
+  background: none;
   display: inline-block;
-  padding: 0 30px;
-  position: relative;
-  text-decoration: none;
-  text-align: center;
-  font-family: arvo;
+  align-items: center;
+  justify-content: space-evenly;
+  height: 100%;
 }
-nav.primary-navigation li a {
+
+nav.primary-navigation li {
+  display: inline-flex;
+  font-family: arvo, Tahoma, Geneva, Verdana, sans-serif;
+  font-size: 16pt;
+  text-align: center;
+  text-transform: none;
+  position: relative;
   color: black;
+  min-width:max-content;
 }
-nav.primary-navigation li a:hover {
-  color: #3ca0e7;
-}
+
 nav.primary-navigation li:hover {
   cursor: pointer;
 }
+
 nav.primary-navigation ul li ul {
   visibility: hidden;
   opacity: 0;
   position: absolute;
-  padding-left: 0;
   left: 0;
-  display: none;
+  margin-top: 1rem;
+  transition: all 0.25s ease;
   background: white;
+  display: inline-block;
+  min-width:max-content;
 }
+
 nav.primary-navigation ul li:hover > ul, nav.primary-navigation ul li ul:hover {
   visibility: visible;
   opacity: 1;
   display: block;
-  min-width: 250px;
+  width: auto;
   text-align: left;
-  padding-top: 20px;
-  box-shadow: 0px 3px 5px -1px #ccc;
+  margin-top: 2rem;
+  background: white;
 }
+
 nav.primary-navigation ul li ul li {
   clear: both;
-  width: 100%;
   text-align: left;
-  margin-bottom: 20px;
+  margin-bottom:0px;
   border-style: none;
+  background: white;
+  display: block;
+}
+
+nav.primary-navigation li a {
+  color: #174ea6;
+  display: block;
+  padding: 8px;
+}
+nav.primary-navigation li a:hover {
+  color: #3ca0e7;
 }
+
 nav.primary-navigation ul li ul li a:hover {
   padding-left: 10px;
   border-left: 2px solid #3ca0e7;
   transition: all 0.3s ease;
+  display: block;
 }
 
+
 .center {
   text-align: center;
 }
 
 a {
   text-decoration: none;
-  color: navy;
+  /* color: navy; */
```

Comment:
I would just remove this line rather than comment it out (we can always add a color specification back in if need be).